Formula/lammps.rb

```diff
@@ -0,0 +1,49 @@
+class Lammps < Formula
+  desc "Molecular Dynamics Simulator"
+  homepage "http://lammps.sandia.gov"
+  url "http://lammps.sandia.gov/tars/lammps-11Aug17.tar.gz"
+  # lammps releases are named after their release date. We transform it to
```

Comment:
It would be nice to make a request upstream that they name their tarballs sanely